### PR TITLE
mingw and msys2 build fix

### PR DESCRIPTION
**Pull request by BilyakA**
exported SHAREDLIBSUFFIXFULLVER variable to mingw-nt build

MSYS2 used to fail build process with `ln: failed to create symbolic link 'libopenh264.': No such file or directory` error.

### CODE REVIEW

**Comment by GuangweiWang** [comment]
thanks for fix.

File: build/platform-mingw_nt.mk [hunk 1]
```diff
@@ -1,7 +1,7 @@
 include $(SRC_PATH)build/x86-common.mk
 SHAREDLIB_DIR = $(PREFIX)/bin
 SHAREDLIBSUFFIX = dll
-SHAREDLIBSUFFIXVER=$(SHAREDLIBSUFFIX)
+SHAREDLIBSUFFIXFULLVER=$(SHAREDLIBSUFFIX)
```
Comment (GuangweiWang) [inline build/platform-mingw_nt.mk]: please also export this variable together:
SHAREDLIBSUFFIXMAJORVER=$(SHAREDLIBSUFFIX)

**Comment by BilyakA** [comment]
Added SHAREDLIBSUFFIXMAJORVER export to mingw-nt build.
I haven't found any changes in build process even with DEBUGSYMBOLS=True flag, but for future changes (for example: lib ver in filename on windows platforms) it may be useful.

**Comment by GuangweiWang** [comment]
@BilyakA yes, you are right.
thanks for the fix!

**Comment by GuangweiWang** [comment]
ship it

**Pull request merged**